code/modules/vore/appearance/sprite_accessories_vr.dm

```diff
@@ -1056,125 +1056,302 @@
+	var/msg_owner_grab_success	= "You pin %prey down onto the floor with your foot and curl your toes up around their body, trapping them inbetween them!"
+
+	//Messages to prey when stepping on/over
+	var/msg_prey_help_walk		= "%owner steps over you carefully!"
+	var/msg_prey_help_run		= "%owner steps over you carefully!"
+	var/msg_prey_harm_walk		= "%owner methodically places their foot upon your body, slowly applying pressure, crushing you against the floor below!"
+	var/msg_prey_harm_run		= "%owner steps carelessly on your body, crushing you!"
+	var/msg_prey_disarm_walk	= "%owner firmly pushes their foot down on you, quite painfully but harmlessly pinning you to the ground!"
+	var/msg_prey_disarm_run		= "%owner pushes you down to the ground with their foot!"
+	var/msg_prey_grab_fail		= "%owner steps down and squishes you with their foot, forcing you down to the ground!"
+	var/msg_prey_grab_success	= "%owner pins you down to the floor with their foot and curls their toes up around your body, trapping you inbetween them!"
+
+	//Messages for smalls moving under larges
+	var/msg_owner_stepunder		= "%owner runs between your legs." //Weird becuase in the case this is used, %owner is the 'bumper' (src)
+	var/msg_prey_stepunder		= "You run between %prey's legs." //Same, inverse
+
 /datum/sprite_accessory/tail/taur/roiz_long_lizard // Not ACTUALLY a taur, but it uses 32x64 so it wouldn't fit in tails.dmi, and having it as a tail bugs up the sprite.
 	name = "Long Lizard Tail (Roiz Lizden)"
 	icon_state = "roiz_tail_s"
 	do_colouration = 0
 	ckeys_allowed = list("spoopylizz")
 
 /datum/sprite_accessory/tail/taur/wolf
-	name = "Wolf"
+	name = "Wolf (Taur)"
 	icon_state = "wolf_s"
 
 /datum/sprite_accessory/tail/taur/wolf/wolf_2c
-	name = "Wolf dual-color"
+	name = "Wolf dual-color (Taur)"
 	icon_state = "wolf_s"
 	extra_overlay = "wolf_markings"
 
 /datum/sprite_accessory/tail/taur/wolf/synthwolf
-	name = "SynthWolf dual-color"
+	name = "SynthWolf dual-color (Taur)"
 	icon_state = "synthwolf_s"
 	extra_overlay = "synthwolf_markings"
 
 /datum/sprite_accessory/tail/taur/naga
-	name = "Naga"
+	name = "Naga (Taur)"
 	icon_state = "naga_s"
 
+	msg_owner_help_walk = "You carefully slither around %prey."
+	msg_prey_help_walk = "%owner's huge tail slithers past beside you!"
+
+	msg_owner_help_run = "You carefully slither around %prey."
+	msg_prey_help_run = "%owner's huge tail slithers past beside you!"
+
+	msg_owner_disarm_run = "Your tail slides over %prey, pushing them down to the ground!"
+	msg_prey_disarm_run = "%owner's tail slides over you, forcing you down to the ground!"
+
+	msg_owner_disarm_walk = "You push down on %prey with your tail, pinning them down under you!"
+	msg_prey_disarm_walk = "%owner pushes down on you with their tail, pinning you down below them!"
+
+	msg_owner_harm_run = "Your heavy tail carelessly slides past %prey, crushing them!"
```

Comment:
Doesn't going "past" someone imply slithering around them (missing) instead of onto them or over them?(hitting/crushing)

Reply:
I think 'past' when applied to sliding can be used that way.

Reply:
Hmm... Still feel like "carelessly sliding over %prey" works better, might be because of English being second language. Up to @Arokha of course.

Reply:
"Sliding" is not the correct verb tense to use in this sentence. "Slides" is.
Substituting "over" for "past" is fine though.

Reply:
I mean, I didn't write any of these messages. If someone wants to "grammarize" them, they can go for it. These are the *existing* taur messages, unaltered, just moved to another file.